app/models/package_manager/pypi.rb

```diff
@@ -13,6 +13,11 @@ class Pypi < Base
     PYPI_PRERELEASE = /(a|b|rc|dev)[0-9]+$/.freeze
     # Adapted from https://peps.python.org/pep-0508/#names
     PEP_508_NAME_REGEX = /^([A-Z0-9][A-Z0-9._-]*[A-Z0-9]|[A-Z0-9])/i.freeze
+    PEP_508_ENVIRONMENT_MARKERS = "python_version|python_full_version|" +
+      "os_name|sys_platform|platform_release|" +
+      "platform_system|platform_version|" +
+      "platform_machine|platform_python_implementation|" +
+      "implementation_name|implementation_version|extra"
```

Comment:
```suggestion
    PEP_508_ENVIRONMENT_MARKERS = %w[
      python_version python_full_version os_name
      sys_platform platform_release platform_system
      platform_version platform_machine platform_python_implementation
      implementation_name implementation_version extra
    ].freeze
    
    PEP_508_ENVIRONMENT_MARKER_REGEX = PEP_508_ENVIRONMENT_MARKERS.join("|")
```

saves you from splitting them back out later